src/WormholeL1VotePool.sol

```diff
@@ -16,24 +17,32 @@ contract WormholeL1VotePool is L1VotePool {
     uint16,
     bytes32
   ) internal {
-    (uint256 proposalId, uint128 against, uint128 inFavor, uint128 abstain) =
+    (uint256 proposalId, uint128 againstVotes, uint128 forVotes, uint128 abstainVotes) =
       abi.decode(payload, (uint256, uint128, uint128, uint128));
+	console2.logString("Begin");
+	console2.logUint(againstVotes);
+	console2.logUint(forVotes);
+	console2.logUint(abstainVotes);
 
     ProposalVote memory existingProposalVote = proposalVotes[proposalId];
+	console2.logUint(existingProposalVote.againstVotes);
+	console2.logUint(existingProposalVote.forVotes);
+	console2.logUint(existingProposalVote.abstainVotes);
+	console2.logString("End");
     if (
-      existingProposalVote.against > against || existingProposalVote.inFavor > inFavor
-        || existingProposalVote.abstain > abstain
+      existingProposalVote.againstVotes > againstVotes || existingProposalVote.forVotes > forVotes
+        || existingProposalVote.abstainVotes > abstainVotes
     ) revert InvalidProposalVote();
 
     // Save proposal vote
-    proposalVotes[proposalId] = ProposalVote(inFavor, against, abstain);
+    proposalVotes[proposalId] = ProposalVote(againstVotes, forVotes, abstainVotes);
 
     _castVote(
       proposalId,
       ProposalVote(
-        inFavor - existingProposalVote.inFavor,
-        against - existingProposalVote.against,
-        abstain - existingProposalVote.abstain
+        forVotes - existingProposalVote.forVotes,
+        againstVotes - existingProposalVote.againstVotes,
+        abstainVotes - existingProposalVote.abstainVotes
```

Comment:
This order was wrong

Reply:
i think it's still wrong, no? against-for-abstain

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/53/commits/627ace93b467b0a0ae1007309dfbaf20e1ff0656